Tutorial_1a_Convolving_Images.ipynb

```diff
@@ -0,0 +1,650 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #2.    smoothed_image = convolve(hdu.data, beam.deconvolve(beam2).as_kernel(pixscale),normalize_kernel=True,)
</pre><p>Block could not be executed because <code>beam</code> , <code>beam2</code> , and  <code>pixscale</code>  are not defined</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Smithsonian/sma-science-tutorials/pull/5/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='27'/>